Sage/carbon
# refactor(radio-button): amend typescript onChange definition - FE-3329

Amend the onChange definition to include HTMLInputElements so that value is present in the Element
change Event.

fixes #3394

### Proposed behaviour

Amend the onChange definition to include `HTMLInputElements` so that `value` is present in the Element
change Event.

### Current behaviour

The onChange definition currently specifies `HTMLElements`. This means that `value` is not present in the Element
change Event.

### Checklist


<del> [ ] Screenshots are included in the PR if useful </del>
<del> [ ] All themes are supported if required </del>
<del> [ ] Unit tests added or updated if required </del>
 <del> [ ] Cypress automation tests added or updated if required </del>
<del> [ ] Storybook added or updated if required </del>

### Additional context
N/A

### Testing instructions
- Open the codesandbox's console. 
- Click on the radio buttons.
- You should be able to see the `value` of the radio button that you have selected outputted in the console. 

https://codesandbox.io/s/brave-shamir-zch2k
